api/envoy/config/listener/v3/listener.proto

```diff
@@ -116,6 +116,10 @@ message Listener {
   // :ref:`FAQ entry <faq_how_to_setup_sni>`.
   repeated FilterChain filter_chains = 3;
 
+  // The default filter chain if none of the filter chain matches. At most 1 filter chain
+  // is allowed here. The filter chain match will be ignored.
+  repeated FilterChain default_filter_chain = 25 [(validate.rules).repeated = {max_items: 1}];
```

Comment:
Why is this repeated vs. just singular?

Reply:
I would like to achieve `0 or 1` default chain.
I don't see we use `optional` tag in proto3 so a `repeated field with max_items = 1` is the closest

Reply:
Just remove the repeated. It's either null or set.